src/get-default-cfg.js

```diff
@@ -9,7 +9,7 @@ module.exports = async () => {
   let migrated = false
 
   const config = {
-    _: 'This is your Now config file. See `now config help`. More: https://git.io/v5ECz'
+    _: 'This is your Now config file. See `now config help`. More: https://zeit.co/docs/features/configuration#now.json'
```

Comment:
Same as the other one. The link you are updating it to is about your project local `now.json` file, _not_ the `~/.now/config.json` file.

Reply:
https://goo.gl/5aRS2s